src/editors/containers/ProblemEditor/index.test.tsx

```diff
@@ -31,76 +25,67 @@ describe('ProblemEditor', () => {
   };
 
   beforeEach(() => {
-    jest.clearAllMocks();
+    initializeMocks();
   });
 
   it('renders Spinner when blockFinished is false', () => {
-    const initialState = {
-      app: { shouldCreateBlock: false },
-      problem: { problemType: 'standard' },
```

Comment:
There were a lot of type errors in this file, because the new `initialState` parameter introduced in https://github.com/openedx/frontend-app-authoring/pull/2326 was not properly typed. For example, `'standard'` is not a valid `problemType`, and `shouldCreateBlock` is a selector, not a state field.